### ISSUE TITLE
Unused import lint does not trigger if imported item is used through another path

### ISSUE THREAD

**Issue by wthrowe** [rust-lang/rust#30078]
This gives no warnings:
```rust
mod a {
    pub fn foo() {}
}

fn main() {
    use a::foo; // Should trigger unused_imports
    a::foo();
}
```

**Comment by Aatch** [comment]
Updated title to be more clear about the issue. I initially read the previous title as "Unused import lint doesn't trigger if import is used", which seems kinda expected :P.

**Comment by wthrowe** [comment]
Thank you.  I agree that the new title is much clearer.
